src/features/chart/redux/actions.ts

```diff
@@ -0,0 +1,45 @@
+import { UPDATE_CHART_RANGE, UPDATE_CHART_DATA } from "./constants";
+import {
+  ActionCreator,
+  Action,
+  ActionWithPayload
+} from "../../../utils/actions";
+import { RESET } from "../../../store/constants";
+
+export interface ChartData {
+  close: number;
+  date: string;
+}
+
+export type ResetAction = Action<RESET>;
```

Comment:
This should be co-located with the `RESET` constant, rather than having to re-define the `ResetAction` type in every feature.